RUNNING_THE_PRODUCT.txt

```diff
@@ -0,0 +1,28 @@
+# How To Run and Contribute To This Java Program
+
+
+##Running The Program (Initial Setup):

```

Comment:
Is **git** actually needed to _run_ the program?
For example, why can't a user just download the zip archive?
This also suggests that the overall heading "How to Run and Contribute To This Java Program" is combining two distinct goals that should be separated in the documentation.